README.md

```diff
@@ -223,6 +223,12 @@ An example of a three server deployment is shown below.  The first server holds
       node.data: true
       node.master: false
       bootstrap.memory_lock: false
+    es_scripts: false
```

Comment:
This variable doesn't exist anywhere else in the playbook. Why did you add it here?